install.md

```diff
@@ -1,20 +1,36 @@
 # Installing Skopeo
 
 ## Distribution Packages
-`skopeo` may already be packaged in your distribution.
+`skopeo` may already be packaged in your distribution. This document lists the
+installation steps for many distros along with their info and support links.
 
 ### Fedora
 
 ```sh
 sudo dnf -y install skopeo
 ```
 
-### RHEL/CentOS ≥ 8 and CentOS Stream
+Package Info: [Dist-Git](https://src.fedoraproject.org/rpms/skopeo) and
```

Comment:
Consider maybe just
```suggestion
[Package Info](https://src.fedoraproject.org/rpms/skopeo) and
```
— the link is useful even for non-git users.

Reply:
I generally prefer spelling out `information` over `info` in formal documentation like this.  That said, `info` is not incorrect.